tools/isobuild/bundler.js

```diff
@@ -2897,6 +2897,11 @@ var writeSiteArchive = Profile("bundler writeSiteArchive", function (
     forceInPlaceBuild: true,
   });
 
+  let sourceRoot;
+  Object.keys(targets).sort().some(key => {
```

Comment:
I'm going to change the `writeSiteArchive` options so that I can just pass in `packageSource.sourceRoot`, which is used for all targets (and is always defined). That said, I do think `files.cwd()` is a reasonable fallback.